CHANGELOG.md

```diff
@@ -17,12 +17,12 @@ Note that this project **does not** adhere to [Semantic Versioning](http://semve
 - We added "Attach file from URL" to right-click context menu to download and store a file with the reference library. [#9646](https://github.com/JabRef/jabref/issues/9646)
 - We enabled updating an existing entry with data from InspireHEP. [#9351](https://github.com/JabRef/jabref/issues/9351)
 - We added a fetcher for the Bibliotheksverbund Bayern (experimental). [#9641](https://github.com/JabRef/jabref/pull/9641)
+- We added BibLatex date formats for parsing dates.
```

Comment:
Please link the issue:

```suggestion
- We added BibLaTeX date formats for parsing dates. [#2753](https://github.com/JabRef/jabref/pull/2753)
```

Reply:
Biblatex has no special casing. It's all lowercase

Reply:
> Biblatex has no special casing. It's all lowercase

Refs https://github.com/koppor/jabref/issues/434